client/me/security-2fa-progress/index.jsx

```diff
@@ -27,14 +27,17 @@ class Security2faProgress extends Component {
 	};
 
 	render() {
+		const doSMSFlow = this.props.doSMSFlow ?? false;
```

Comment:
Can't we just destructure from `this.props` here?

```suggestion
		const { doSMSFlow } = this.props;
```